projects/plugins/boost/app/admin/class-admin.php

```diff
@@ -128,7 +128,12 @@ public function enqueue_scripts() {
 		wp_localize_script(
 			$admin_js_handle,
 			'Jetpack_Boost',
-			$this->config->constants()
+			array_merge(
+				$this->config->constants(),
+				array(
+					'siteCustomPostTypes' => $this->get_custom_post_types(),
+				)
```

Comment:
Is there a reason not to just add this in `Config::constants`?

Reply:
Hm, not sure why I didn't add it there - makes the most sense. Done in [e89357e](https://github.com/Automattic/jetpack/pull/31811/commits/e89357e8c93ad9dc1956b4178c6afa095ff4828f).